app/templates/profile.hbs

```diff
@@ -1,17 +1,43 @@
-<button {{action 'signOut'}}>Sign Out</button>
-<button {{action 'saveProfile'}}>Save Profile</button>
-
-{{#each model as |user|}}
-  <article class="profile">
-    <h3>{{user.name}}</h3>
-    <div class="detail email">
-      <span>Email:</span> {{user.email}}
-    </div>
-    <div class="detail city">
-      <span>Location:</span> {{user.city}}, {{user.state}}
-    </div>
-    <div class="detail location">
-      <span>Radius:</span> {{user.radius}} miles
-    </div>
-  </article>
+<div class="div-outer-white">
```

Comment:
Also maybe you could add indentation where appropriate here as well? Just for readability